api/XRPermissionDescriptor.json

```diff
@@ -1,200 +0,0 @@
-{
-  "api": {
-    "XRPermissionDescriptor": {
-      "__compat": {
-        "mdn_url": "https://developer.mozilla.org/docs/Web/API/XRPermissionDescriptor",
```

Comment:
https://developer.mozilla.org/docs/Web/API/XRPermissionDescriptor will need to be removed.